python/rascaline/utils.py

```diff
@@ -20,3 +22,225 @@ def _call_with_growing_buffer(callback, initial=1024):
+            "interaction_cutoff",
+            "inversion_symmetry",
+            "max_angular",
+            "max_radial",
+            "normalize",
+            "optimization_args",
+            "optimization",
+            "radial_basis",
+            "soap_type",
+        ]
+
+        if any([key not in anticipated_hypers for key in hyperparameters]):
+            raise ValueError(
+                "I do not know what to do with the following hyperparameter entries:\n\t".format(
+                    "\n\t".join(
+                        [
+                            key
+                            for key in hyperparameters
+                            if key not in anticipated_hypers
+                        ]
+                    )
+                )
+            )
+
+        new_hypers["atomic_gaussian_width"] = hyperparameters.pop(
+            "gaussian_sigma_constant", None
+        )
+        new_hypers["max_angular"] = hyperparameters.pop("max_angular", None)
+        new_hypers["max_radial"] = hyperparameters.pop("max_radial", None)
+        new_hypers["cutoff"] = hyperparameters.pop("interaction_cutoff", None)
+
+        if  'radial_basis' in hyperparameters:
+            new_hypers["radial_basis"] = {hyperparameters.pop("radial_basis").title(): {}}
+            if new_hypers["radial_basis"] != "Gto":
+                warnings.warn("WARNING: rascaline currently only supports a Gto basis.")
+
+        if hyperparameters.get("cutoff_function_type", None) == "ShiftedCosine":
+            new_hypers["cutoff_function"] = {
+                hyperparameters.pop("cutoff_function_type", None): {
+                    "width": hyperparameters.pop("cutoff_smooth_width", None)
+                }
+            }
+        else:
+            new_hypers["cutoff_function"] = {"Step": {}}
+            if hyperparameters.get("cutoff_function_type", None) == "RadialScaling":
+                params = hyperparameters.pop("cutoff_function_parameters", None)
+                new_hypers["radial_scaling"] = {
+                    "Willatt2018": {
+                        "exponent": int(params.get("exponent", None)),
+                        "rate": params.get("rate", None),
+                        "scale": params.get("scale", None),
+                    }
+                }
+
+        deprecated_params = [
+            "global_species",
+            "expansion_by_species_method",
+            "soap_type",
+            "compute_gradients",
+        ]
+        if any([d in hyperparameters for d in deprecated_params]):
+            warnings.warn(
+                "{} are not required parameters in the rascaline software infrastructure".format(
+                    ",".join(
+                        [f"`{d}`" for d in deprecated_params if d in hyperparameters]
+                    )
+                )
+            )
+
+        not_supported = [
+            "coefficient_subselection",
+            "covariant_lambda",
+            "gaussian_sigma_type",
```

Comment:
`gaussian_sigma_type="Constant"` (which is the only one implemented by librascal) is also what rascaline is doing by default